### PR TITLE
feat(node): make toleration k8s_arch configurable

### PR DESCRIPTION
**Pull request by verbotenj**

### CODE REVIEW

File: stage3/nodes.tf [hunk 1]
```diff
@@ -73,7 +73,7 @@ module "ext_cardano_node" {
           effect   = "NoSchedule"
           key      = "kubernetes.io/arch"
           operator = "Equal"
-          value    = "amd64"
+          value    = "arm64"
```
Comment (agaffney) [inline stage3/nodes.tf:76]: Do we want to hardcode a specific architecture here? We could have tolerations for both of them

File: stage3/variables.tf [hunk 2]
```diff
@@ -60,6 +65,11 @@ variable "storage_class_name_preprod" {
   default     = "pd-ssd"
 }
 
+variable "toleration_k8s_arch_preprod" {
+  description = "toleration for the k8s arch"
+  default     = "x86"
```
Comment (agaffney) [inline stage3/variables.tf:70]: amd64

File: stage3/variables.tf [hunk 3]
```diff
@@ -70,6 +80,11 @@ variable "storage_class_name_preview" {
   default     = "pd-ssd"
 }
 
+variable "toleration_k8s_arch_preview" {
+  description = "toleration for the k8s arch"
+  default     = "x86"
```
Comment (agaffney) [inline stage3/variables.tf:85]: amd64

File: stage3/variables.tf [hunk 4]
```diff
@@ -50,6 +50,11 @@ variable "storage_class_name_mainnet" {
   default     = "pd-ssd"
 }
 
+variable "toleration_k8s_arch_mainnet" {
+  description = "toleration for the k8s arch"
+  default     = "x86"
```
Comment (agaffney) [inline stage3/variables.tf:55]: amd64

**Review by wolf31o2 — APPROVED** [review]

**Pull request merged**